packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.base.tsx

```diff
@@ -208,31 +207,41 @@ export class DropdownBase extends React.Component<IDropdownInternalProps, IDropd
     const onRenderPlaceholder = props.onRenderPlaceholder || props.onRenderPlaceHolder || this._onRenderPlaceholder;
 
     const selectedOptions = getAllSelectedOptions(options, selectedIndices);
+    const selectedOptionsAriaLabel = this._joinSelectedOptions(selectedOptions);
     const divProps = getNativeProps(props, divProperties);
 
     const disabled = this._isDisabled();
 
     const optionId = id + '-option';
-    const ariaAttrs =
-      multiSelect || disabled
-        ? {
-            role: undefined,
-            ariaActiveDescendant: undefined,
-            childRole: undefined,
-            ariaSetSize: undefined,
-            ariaPosInSet: undefined,
-            ariaSelected: undefined
-          }
-        : // single select
-          {
-            role: 'listbox',
-            ariaActiveDescendant:
-              isOpen && selectedIndices.length === 1 && selectedIndices[0] >= 0 ? this._id + '-list' + selectedIndices[0] : optionId,
-            childRole: 'option',
-            ariaSetSize: this._sizePosCache.optionSetSize,
-            ariaPosInSet: this._sizePosCache.positionInSet(selectedIndices[0]),
-            ariaSelected: selectedIndices[0] === undefined ? undefined : true
-          };
+    const ariaAttrs = disabled
+      ? {
+          role: undefined,
+          ariaActiveDescendant: undefined,
+          childRole: undefined,
+          ariaSetSize: undefined,
+          ariaPosInSet: undefined,
+          ariaSelected: undefined
+        }
+      : multiSelect
+      ? {
+          role: 'listbox',
+          ariaActiveDescendant:
+            isOpen && selectedIndices.length === 1 && selectedIndices[0] >= 0 ? this._id + '-list' + selectedIndices[0] : optionId,
+          childRole: 'option',
+          ariaSetSize: this._sizePosCache.optionSetSize,
```

Comment:
""Apple, Banana, Grape selected, selection contains 7 items" - doesn't seem right.. wouldn't the selection have only 3 items? I think ariaSetSize should be undefined here

Reply:
I'm curious if there are any problems you noticed if you revert line 216-244.. wondering if these changes are necessary (naively without testing out, I would assume keeping line 210, 294 and 456-467 are good enough to fix the bug)

Reply:
Set size seems to be the total amount of options in the dropdown. For single select dropdowns, it reads out "selection contains 7 items" as well.

Reply:
From my testing, we don't achieve the proper readout without these attributes combined. I can double check to see if there are any extras I've added that we don't need.

Reply:
Alright so just double checked, and everything is needed except for `ariaActiveDescendant`. Without `role`, `childRole`, and `ariaSelected`, it does not read out the selected options in a multi-select dropdown.

Reply:
size is needed in single select for a different reason. Let me test it out/sync with you

Reply:
Hey @natalieethell, I'll need more time to close on this. I'll look into this tomorrow and touch base with you :)